code/datums/diseases/advance/symptoms/gravity.dm

```diff
@@ -0,0 +1,97 @@
+		bodies = list("Poltergeist")
+	if(A.resistance >= 6)
+		severity += 1
+		prefixes = list("Gravitic ", "Crushing ")
+	if(A.resistance >= 10)
+		severity += 2
+		prefixes = list("Gibbington's ", "Crushing ", "Gravitic ")
+
+/datum/symptom/gravity/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.transmission >= 6)
+		throwobj = TRUE
+	if(A.resistance >= 6)
+		power = 4
+	if(A.resistance >= 10)
+		power = 6
+
+/datum/symptom/gravity/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	if(A.stage == 5)
+		if(!gravcomponent)
+			gravcomponent = M.AddComponent(/datum/component/forced_gravity, rand(0, power))
+		else 
+			var/lastgravity = gravcomponent.gravity
+			var/realpower = power
+			if(lastgravity >= 3)
+				realpower = 2 //never have a damaging cycle twice in a row
+			gravcomponent.gravity = rand(0, power)
+			M.update_gravity(gravcomponent.gravity)
+			if(throwobj && prob(40))
+				var/list/throwing = list()
+				for(var/atom/movable/AM in orange(5, get_turf(M)))
+					if(AM == M || AM.anchored)
+						continue
+					if(ismob(AM))
+						var/mob/L = AM
+						if(L.mob_negates_gravity())
+							continue
+					throwing += AM
+				for(var/O in 0 to min((power + 1), LAZYLEN(throwing)))
+					var/atom/movable/AM = pick_n_take(throwing)
+					if(AM)
+						AM.throw_at(M, 4, 2)
+			if(gravcomponent.gravity >= 6 && M.stat >= UNCONSCIOUS && !HAS_TRAIT(M, TRAIT_NODISMEMBER))
+				M.visible_message("<span class='userdanger'>[M]'s body contorts, compresses, and collapses in on itself, before exploding into a shower of gore!</span>")
+				playsound(M, 'sound/magic/demon_consume.ogg', 50, 1, -1)
+				M.gib()
+				return
+			if((gravcomponent.gravity - realpower) >= 2 && (M.mobility_flags & MOBILITY_STAND) && !(isspaceturf(get_turf(M))))
+				M.Knockdown((gravcomponent.gravity - realpower) * 10)
+				playsound(M, 'sound/effects/meteorimpact.ogg', 25, 1, -1)
+				M.visible_message("<span class='danger'>[M] is slammed into the ground by an unseen force!</span>",   "<span class='userdanger'>A sudden increase in gravity slams you into the ground!</span>")
+			if(gravcomponent.gravity == 0)
+				M.visible_message("<span class='danger'>[M] floats away from the ground, as if gravity no longer works on them!</span>",   "<span class='userdanger'>The ground falls away under you... You're floating!</span>")
+	else if(prob(40))
+		to_chat(M, "<span class='danger'>[pick("You feel heavy", "You feel incredibly light", "Your arms sag towards the floor")]</span>")
+
+/datum/symptom/gravity/End(datum/disease/advance/A)
+	. = ..()
+	if(gravcomponent)
+		gravcomponent.RemoveComponent()
+		gravcomponent = null
+
+/datum/symptom/gravity/OnDeath(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/carbon/M = A.affected_mob
+	if(power >= 6)
+		M.visible_message("<span class='userdanger'>[M]'s body contorts, compresses, and collapses in on itself, before exploding into a shower of gore!</span>")
+		M.gib()
```

Comment:
I believe this will cause `gib` to be called twice if the mob is killed by the disease's resistance 10 threshold, which might have unintended side effects. I'm not sure if that'll really be an issue, but it could cause a runtime when the mob tries to continue dying after being qdel'd.